test/theme/hs_warning_bottom_sheet_test.dart

```diff
@@ -0,0 +1,80 @@
+import 'package:flutter/material.dart';
+import 'package:flutter/widgets.dart';
+import 'package:flutter_svg/svg.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:hatspace/gen/assets.gen.dart';
+import 'package:hatspace/theme/hs_theme.dart';
+import 'package:hatspace/theme/widgets/hs_buttons.dart';
+import 'package:hatspace/theme/widgets/hs_warning_bottom_sheet.dart';
+
+import '../widget_tester_extension.dart';
+
+void main() {
```

Comment:
add one more test case: HsWarningBottomSheetView() -> throw exception require at least .....

Reply:
oops. it's covered